nixos/modules/services/video/epgstation/default.nix

```diff
@@ -0,0 +1,277 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.epgstation;
+
+  username = config.users.users.epgstation.name;
+  groupname = config.users.users.epgstation.group;
+
+  settingsFmt = pkgs.formats.json {};
```

Comment:
:mag: not sure this local variable is getting us much here, given that the filename contains `.json` so we wouldn't really ever be able to change it.

Reply:
`settingsFmt` is used to generate JSON output from NixOS options. https://github.com/NixOS/nixpkgs/pull/75584

Reply:
yes, my point is that having it as a local variable doesn't feel necessary - we could just make the next line 
```nix
 settingsTemplate = (pkgs.formats.json {}).generate "config.json" cfg.settings;
```

Reply:
Sorry, I misunderstood your point. I still think it should be kept as a local variable because `settingsFmt` is used twice.
```nix
settingsTemplate = settingsFmt.generate "config.json" cfg.settings;
```
```nix
freeformType = settingsFmt.type;
```

Reply:
oh, oops, missed the second usage. Disregard!